pandas/core/indexes/accessors.py

```diff
@@ -126,6 +126,32 @@ class DatetimeProperties(Properties):
     """
 
     def to_pydatetime(self):
+        """
+        Return DatetimeIndex as object ndarray of datetime.datetime objects.
+
+        This function converts the Pandas Series DatetimeIndex to
+        a ndarray object. These two types are different and hence
+        2 differnt functions.
+
+        Returns
+        -------
+        datetimes : ndarray
+
+        See Also
+        --------
+        pd.DatetimeIndex.to_pydatetime : Convert a Timestamp object to a native
+            Python datetime object.
+        pd.date.dt.values.to_datetime : For an Index containing strings or
+            datetime.datetime objects, attempt conversion to DatetimeIndex.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'date': [pd.to_datetime('2018-03-10'), pd.to_datetime('2017-02-01')]})
```

Comment:
make a series like: ``In [1]: Series(pd.date_range('20180310', periods=2))``
show its return, then show the return of using ``.dt.to_pytdatetime()``

Reply:
Fixed.